README.md

```diff
@@ -4,8 +4,84 @@
 
 **Standard Exit Format for L2s built on EVM chains**
 
-An exit format allows calldata to specify how assets locked up and redistributed in an L2 should be paid out in L1. We present a general standard for such a format, along with coders written in Typescript and Cairo, which will aid L2 interoperability and support arbitrary tokens
+We present a general standard for such a format, along with coders written in Typescript and Cairo, which will aid L2 interoperability and support arbitrary tokens
 
-Ethereum is a multi-asset blockchain.
-Many applications require multiple assets to be frozen and liquidated in a smart contract, possibly atomically.
-Having a standard for this will make interoperability much easier. In the statechannels.org project, we have an existing format for exits (which is currently called an 'outcome'), expressed both in solidity and typescript. As well as coders, we require functions that transform the exit according to set rules: for example to execute a part of the exit, paying out one party and generating a new exit with that party removed. We will explore the use of zero knowledge methods to update exits stored on chain more efficiently. We will improve and generalize our existing format in order to save gas, reduce complexity and handle custom exit logic. We will present a standard exit format which other state channel and rollup protocols can conform with.
+The idea behind this library is to standardise the data structures used in exiting a layer 2 system: whether that is a Connext state channel or a rollup such as Arbitrum or optimism. An exit format allows one to specify how assets locked up and redistributed in an L2 should be paid out in L1.  Our hope is that as many layer 2 projects as possible can contribute to this repo and consider adopting the format in their system. If that happens, it should enable greater interoperability between these layer 2s, which otherwise are a little bit silo-ed.
```

Comment:
```suggestion
The idea behind this library is to standardise the data structures used in exiting a layer 2 system: whether that is a [Connext](https://connext.network/) state channel or a rollup such as [Arbitrum](https://offchainlabs.com/) or [Optimism](https://optimism.io/). An exit format allows one to specify how assets locked up and redistributed in an L2 should be paid out in L1.  Our hope is that as many layer 2 projects as possible can contribute to this repo and consider adopting the format in their system. If that happens, it should enable greater interoperability between these layer 2s, which otherwise are a little bit silo-ed.
```